Sources/Tools/plSystemInfo/main.cpp

```diff
@@ -41,12 +41,15 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 *==LICENSE==*/
 
 #include "hsSystemInfo.h"
+#include "plFileSystem.h"
```

Comment:
```suggestion
#include "plFileSystem.h"
#include "hsSystemInfo.h"
```

Alphabetize core includes

Reply:
This suggestion is still relevant.